src/site/stages/build/process-cms-exports/transformers/node-health_care_region_detail_page.js

```diff
@@ -0,0 +1,51 @@
+const {
+  getDrupalValue,
+  utcToEpochTime,
+  createMetaTagArray,
+} = require('./helpers');
+
+const transform = entity => ({
+  entityType: 'node',
+  entityBundle: 'health_care_region_detail_page',
+  title: getDrupalValue(entity.title),
+  changed: utcToEpochTime(getDrupalValue(entity.changed)),
+  entityPublished: getDrupalValue(entity.moderationState) === 'published',
+  entityMetatags: createMetaTagArray(entity.metatag.value),
+  entityUrl: {
+    path: entity.path[0].alias,
+  },
+  fieldAlert: getDrupalValue(entity.fieldAlert),
+  fieldContentBlock: entity.fieldContentBlock,
+  fieldFeaturedContent: entity.fieldFeaturedContent,
+  fieldIntroText: getDrupalValue(entity.fieldIntroText),
+  fieldOffice: {
+    entity: {
+      entityLabel: entity.fieldOffice[0].entity.entityLabel,
+      fieldNicknameForThisFacility:
+        entity.fieldOffice[0].entity.fieldNicknameForThisFacility,
+      title: entity.fieldOffice[0].entity.title,
+    },
+  },
+  fieldRelatedLinks:
```

Comment:
Like I said above, we can probably get away with `fieldOffice: entityFieldOffice[0] || null,`.

Honestly, it isn't the end of the world either way, but I think the less clever we can be about it, the better off we'll be down the road.